Polyhedron/demo/Polyhedron/Plugins/IO/GOCAD_io_plugin.cpp

```diff
@@ -96,7 +96,7 @@ bool Polyhedron_demo_gocad_plugin::save(const CGAL::Three::Scene_item* item, QFi
     return false;
 
   std::ofstream out(fileinfo.filePath().toUtf8());
-
+  out.precision (std::numeric_limits<double>::digits10 + 1);
   Polyhedron* poly = const_cast<Polyhedron*>(poly_item->polyhedron());
```

Comment:
It should be `+ 2` instead of `+ 1`.
See:
* http://en.cppreference.com/w/cpp/types/numeric_limits/digits10
* http://en.cppreference.com/w/cpp/types/numeric_limits/max_digits10

For `double`, `digits10` is 15, and `max_digits10` is 17.